docs/deprecations.rst

```diff
@@ -59,6 +59,20 @@ another mode before saving::
     im = Image.new("I", (1, 1))
     im.convert("I;16").save("out.png")
 
+ImageCms.ImageCmsProfile.product_name and .product_info
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. deprecated:: 12.0.0
+
+``ImageCms.ImageCmsProfile.product_name`` and the corresponding
+``.product_info`` attributes have been deprecated, and will be removed in
+Pillow 13 (2026-10-15). These attributes can be accessed on the ``.profile``
+attribute of ``ImageCmsProfile`` instead.
+
+Note that ``.product_name`` and ``.product_info`` have been set to ``None`` on
+``ImageCmsProfile`` since Pillow 2.3.0 (2014-01-01), so any working code that
+makes use of this data will already access it on ``.profile``.
```

Comment:
```suggestion
Pillow 13 (2026-10-15). They have been set to ``None`` since Pillow 2.3.0.
```

If these attributes don't have a direct mapping,
```pycon
>>> from PIL import ImageCms
>>> profile = ImageCms.ImageCmsProfile(ImageCms.createProfile("sRGB"))
>>> profile.profile.product_name
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'PIL.ImageCms.core.CmsProfile' object has no attribute 'product_name'
```
then I think it is sufficient to just explain that they've been `None` for quite some time.